docs/platform_management_plan/software_verification.rst

```diff
@@ -468,6 +486,10 @@ of test based on `pytest <https://pytest.org>`__.
 The platform developed in this project supports `Rust <https://www.rust-lang.org>`__ as a programming language. Its
 built-in test framework is used to test respective software components.
 
+.. rubric:: Test Scenario Framework
+
+Supports Component Integration Tests by providing scenarios in C++ and Rust that are tested with single Test Case implementation in Pytest.
```

Comment:
Is the sentence properly starting with `Supports`? Do you want to say that the `Test Scenario Framework supports`